src/test/java/org/folio/bulkops/service/BulkOperationServiceTest.java

```diff
@@ -601,7 +601,7 @@ void  shouldPopulateErrorToBulkOperationIfS3IssuesForConfirmChangesForInstanceMa
         .thenReturn(BulkOperationDataProcessing.builder()
```

Comment:
Given the number of places in the code where changes have been made, it is essential to add or enhance existing tests to emulate the occurrence of both S3 exceptions and errors arising from other causes.